### DISCUSSION TITLE
fastapi-users SQLAlchemy 2.X.X support?

### DISCUSSION THREAD

**Discussion by michaelburris** [fastapi-users/fastapi-users#1158] [Q&A] [answered]
I tried updating a fastapi-users project to SQLAlchemy 2.0.1 and it throw an exception starting with "Type annotation for "User.id" can't be correctly interpreted for".

It had an issue with the fast api BaseUser.

**Comment by frankie567** [comment, marked as answer]
Yes, that's something I plan to do in the coming weeks. For now, you should stick on 1.4

**Reply by michaelburris** [reply]
Thanks you!

**Comment by frankie567** [comment]
SQLAlchemy 2.0 support is now available in [10.4](https://github.com/fastapi-users/fastapi-users/releases/tag/v10.4.0) 😄
